pkg/cmd/build/build.go

```diff
@@ -246,6 +246,33 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 		file = extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName)
 	}
 
+	defaultBuildArgs := map[string]string{
+		model.OktetoContextEnvVar:         okteto.Context().Name,
+		model.OktetoUserNameEnvVar:        okteto.Context().Username,
+		model.OktetoTokenEnvVar:           okteto.Context().Token,
```

Comment:
I'm not sure if the okteto token should be added. If it is used, an image would be built with that token inside. 

I think having only `model.OktetoContextEnvVar` and `model.OktetoNamespaceEnvVar` should be ok unless you need also the others, but I understood that you only need the namespace, right? @odedlaz

Reply:
@ifbyol, you're correct. I was trying to go the extra mile, probably went too far :) removed everything except `OktetoContextEnvVar` and `OktetoNamespaceEnvVar`. Should be easy to extend in the future.